sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteReview.md

```diff
@@ -0,0 +1,117 @@
+﻿---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pvrk
+ms.author: pvrk
+manager: 
+ms.reviewer:
+---
+
+# Get-SPOSiteReview
+
+## SYNOPSIS
+Track all site access reviews initiated by all SharePoint admins
```

Comment:
>SharePoint admins [](http://example.com/codeflow?start=47&length=17)

do you mean "SharePoint Administrator" specifically, or admins in general?

Sentences must end with `.`